activesupport/lib/active_support/core_ext/date/calculations.rb

```diff
@@ -85,6 +85,11 @@ def end_of_day
   end
   alias :at_end_of_day :end_of_day
 
+  # Returns a Range representing the whole day of the current date.
```

Comment:
Don't love the phrasing "current date", but this is based on the docs for the Time method, which say "current time". In both cases, we mean the instance's date/time.